src/main/java/at/hannibal2/skyhanni/features/misc/ThunderBottleNotification.kt

```diff
@@ -0,0 +1,32 @@
+package at.hannibal2.skyhanni.features.misc
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.events.LorenzChatEvent
+import at.hannibal2.skyhanni.skyhannimodule.SkyHanniModule
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.RegexUtils.matches
+import at.hannibal2.skyhanni.utils.repopatterns.RepoPattern
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+import kotlin.time.Duration.Companion.seconds
+
+@SkyHanniModule
+object ThunderBottleNotification {
+
+    private val config get() = SkyHanniMod.feature.misc
+
+    private val thunderBottleChargedPattern by RepoPattern.pattern(
+        "bottle.charged",
```

Comment:
Better name, since that one is very likely to have a conflict in the future
```suggestion
        "thunderbottle.charged",
```